### ISSUE TITLE
Access-Control-Allow-Credentials: improvement when setting its value

### ISSUE THREAD

**Issue by mhagnumdw** [pac4j/pac4j#1185]
According to the documentation [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials#Directives), replicated below

> The only valid value for this header is true (case-sensitive). If you don't need credentials, omit this header entirely (rather than setting its value to false).

The following code snippet in the [CorsAuthorizer](https://github.com/pac4j/pac4j/blob/master/pac4j-core/src/main/java/org/pac4j/core/authorization/authorizer/CorsAuthorizer.java#L46) class should look like this
```java
if (allowCredentials != null && allowCredentials) {
```
instead of
```java
if (allowCredentials != null) {
```
What do you think?

**Comment by leleuj** [comment]
It seems so. An existing value is not enough, it must be true. Would you mind submitting a PR for that? Thanks

**Comment by leleuj** [comment]
Not necessary anymore. Fixed before the 3.2.0 release.
